components/tidb_query/src/rpn_expr/impl_time.rs

```diff
@@ -128,13 +128,32 @@ pub fn year(ctx: &mut EvalContext, t: &Option<DateTime>) -> Result<Option<Int>>
     Ok(Some(Int::from(t.year())))
 }
 
+#[rpn_fn(capture = [ctx])]
+#[inline]
+pub fn day_name(ctx: &mut EvalContext, t: &Option<DateTime>) -> Result<Option<Bytes>> {
+    match t {
+        Some(t) => {
+            if t.invalid_zero() {
+                return ctx
+                    .handle_invalid_time_error(Error::incorrect_datetime_value(t))
+                    .map(|_| Ok(None))?;
+            }
+            use crate::codec::mysql::time::WeekdayExtension;
```

Comment:
Would you mind move this import to the top of the file?

Reply:
I have modified it